analyses/snv-callers/run_caller_consensus_analysis.sh

```diff
@@ -45,15 +45,36 @@ Rscript analyses/snv-callers/scripts/02-merge_callers.R \
 python3 analyses/snv-callers/scripts/01-setup_db.py \
   --db-file $dbfile \
   --consensus-file $consensus_file
-  
+
+###################### Create intersection BED files ###########################
+# Make All mutations BED file
+bedtools intersect \
+  -a data/WGS.hg38.strelka2.unpadded.bed \
+  -b data/WGS.hg38.mutect2.unpadded.bed > scratch/intersect_strelka_mutect_WGS.bed
```

Comment:
It probably makes sense to make these scratch file names bash variables, for ease of reuse and error prevention.

Reply:
Will do